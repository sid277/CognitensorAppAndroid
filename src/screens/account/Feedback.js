import React from 'react';
import { Text } from 'react-native';
import DefaultView from '../../components/default/DefaultView';

const Feedback = () => {
  return (
    <DefaultView>
      <Text>Feedback Screen</Text>
    </DefaultView>
  );
};

export default Feedback;
