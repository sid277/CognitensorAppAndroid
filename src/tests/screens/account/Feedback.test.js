import React from 'react';
import { shallow } from 'enzyme';
import Feedback from '../../../screens/account/Feedback';
import renderer from 'react-test-renderer';


test('should test Feedback component', () => {
const wrapper = shallow(<Feedback />);
expect(wrapper).toMatchSnapshot();
});