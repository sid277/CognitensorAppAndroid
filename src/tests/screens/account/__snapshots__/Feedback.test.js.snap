// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test Feedback component 1`] = `ShallowWrapper {}`;
